openstreetmap-carto.style

```diff
@@ -41,6 +41,7 @@ node,way   railway      text         linear
 node,way   ref          text         linear
 node,way   religion     text         linear
 way        route        text         linear
+way        roller_coaster      text  linear
```

Comment:
It doesn't need its own column

Reply:
could you explain how i would do this instead?